CHANGELOG.asciidoc

```diff
@@ -1,4 +1,6 @@
 include::./changelogs/head.asciidoc[]
```

Comment:
Docs don't build because this `head.asciidoc` no longer exists. Remove this line
```suggestion
```